### PR TITLE
Release V12 data

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose/implementation Section
#326 - Release V12 data

### release-v12-20191217
- release date: 2019-12-17
- status: available
- changes:
  - Add `data-file-descriptions.md` with data release to better track file types, origins, and workflows per [#334](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/334) and [#336](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/336)
  - Add stranded RNA-Seq for 23 PNOC samples and 21 CBTTC samples previously sequenced using a polyA library prep. Files updated:
    - pbta-fusion-arriba.tsv.gz
    - pbta-fusion-starfusion.tsv.gz
    - pbta-gene-expression-rsem-tpm.stranded.rds
    - pbta-gene-expression-rsem-fpkm.stranded.rds
    - pbta-isoform-expression-rsem-tpm.stranded.rds
    - pbta-isoform-counts-rsem-expected_count.stranded.rds
    - pbta-gene-counts-rsem-expected_count.stranded.rds
    - pbta-gene-expression-kallisto.stranded.rds
    - pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds
  - Add recurrently-fused genes by histology and matrix of recurrently-fused genes by biospecimen from [fusion filtering and prioritization analysis](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering)
  - Update consensus TMB files and MAF [#333](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/333)
  - Add RNA-Seq [collapsed matrices](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/287) - wrong files (tables of transcripts removed) were included with [V10](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/273)
  - Rename `WGS.hg38.mutect2.unpadded.bed` to `WGS.hg38.mutect2.vardict.unpadded.bed` to better reflect usage
  - Update `pbta-histologies.tsv` to add new RNA-Seq samples listed above, [#222 harmonize disease separators](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/222), and reran [medulloblastoma classifier](https://github.com/d3b-center/medullo-classifier-package) using V12 RSEM fpkm collapsed files
    - BS_2Z1MKS84, BS_5VQP0E6K re-classified from Group4 to WNT and BS_3BDAG9YN, BS_8T7DZV2F, and BS_JTMXAMB7 re-classified from Group3 to WNT
  - Add CNVkit GISTIC results focal CN analyses, eg: [#244](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/244) and [#8](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/8)

#### Which areas should receive a particularly close look?
please make sure I didn't break anything :)

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Not yet - awaiting a few files to be transferred to S3

### CODE REVIEW

**Comment by jharenza** [comment]
ok @jaclyn-taroni not sure why the last 3 rows don't have dividers and look like one row...

**Comment by migbro** [comment]
Files uploaded to bucket https://s3.console.aws.amazon.com/s3/buckets/kf-openaccess-us-east-1-prd-pbta/data/release-v12-20191217/?region=us-east-1&tab=overview, with md5sums also uploaded.

**Comment by jashapiro** [comment]
Download seemed to work at my end!
I did look through the changes to disease types related to #222, and found a few lines that I had questions about: https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/222#issuecomment-566837893 If those do need to be addressed, it would be good to get them in this release.

File: doc/data-files-description.md [hunk 1]
```diff
@@ -53,10 +53,14 @@ This document contains information about all data files associated with this pro
...
 |`pbta-isoform-expression-rsem-tpm.stranded.rds` | PBTA data file | [Gene expression abundance estimation](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#gene-expression-abundance-estimation); [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/master/workflow/kfdrc_RNAseq_workflow.cwl) | Gene expression - RSEM TPM for stranded samples (transcript-level)
-|`pbta-snv-consensus-mutation-tmb.tsv` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Tumor mutation burden statistics calculated from consensus SNV, using Strelka2 counts and BED window sizes
+|`pbta-snv-consensus-mutation-tmb-all.tsv` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Tumor mutation burden statistics calculated from consensus SNV, using Strelka2 counts and BED window sizes
```
Comment (jaclyn-taroni) [inline doc/data-files-description.md]: @cansavvy can you reply to this comment with what you would say for the file descriptions for the all TMB and coding TMB files please? I will also port that information over to `doc/data-formats.md`.

Comment (cansavvy) [reply doc/data-files-description.md]: For all: ```Tumor mutation burden statistics calculated from Strelka2 and Mutect2 SNV consensus, and the intersection of Strelka2 and Mutect2 BED windows sizes.```
For coding: ```Coding only tumor mutation burden statistics calculated from the number of exonic Strelka2, Mutect2, and Lancet consensus SNVs and size of the intersection of all three callers' BED windows and the Gencode v27 exome. ```

**Comment by jaclyn-taroni** [comment]
@jharenza the changes I made in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/347/commits/3a3ab779259be77b17e8bac222aa10e374006042 should get ported over to the `data-file-description.md` file that's on S3

**Comment by jharenza** [comment]
> Download seemed to work at my end!
> I did look through the changes to disease types related to #222, and found a few lines that I had questions about: [#222 (comment)](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/222#issuecomment-566837893) If those do need to be addressed, it would be good to get them in this release.

@jashapiro let me look into this

**Comment by jaclyn-taroni** [comment]
Also tried to be more specific about one of the new fusion tables https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/347/commits/aba9d20444e51b81b68b1a04ae333c98c3bffa5e#diff-0468bf9add309a2a34494766546a847aR66

**Comment by jaclyn-taroni** [comment]
I was under the impression that we wanted to rename the `CHANGELOG.md` file in the data download file to `release-notes.md` to match what is included in the repository, is that correct?

**Comment by jharenza** [comment]
@jaclyn-taroni @jashapiro - @migbro is working on updating the files now and we will keep you posted once they are copied.

**Comment by jaclyn-taroni** [comment]
👍  thanks! I'm getting started on the req'd changes for #337 now.

**Comment by jharenza** [comment]
actually @jaclyn-taroni we are changing it now...

**Comment by jaclyn-taroni** [comment]
Moving the comment here https://github.com/AlexsLemonade/OpenPBTA-analysis/commit/aba9d20444e51b81b68b1a04ae333c98c3bffa5e#commitcomment-36486645 into this thread

> @jaclyn-taroni - I kept CHANGELOG.md in the data download since that was what we used in the past. We can standardize in v13 if we want to change?

We are changing a number of other things now that will affect the CI file generation (e.g., including `data-file-description.md`) so I don't see the rationale for waiting.

**Comment by jaclyn-taroni** [comment]
:+1: I did not get this
> actually @jaclyn-taroni we are changing it now... 

before I posted !

**Comment by migbro** [comment]
these files have been updated, as well as the md5sum file:
```
94960674ba9f2bec71b36e44a6c4e9bf  pbta-fusion-recurrently-fused-genes-byhistology.tsv
ca929840f6f9347899165c6f21645f8f  pbta-fusion-recurrently-fused-genes-bysample.tsv
4cf650557d8bbbb6fc5456b30990f5d4  pbta-histologies.tsv
```

**Comment by jharenza** [comment]
we just updated the clinical `dd7940312421ef138eb85f843a701cee pbta-histologies.tsv` and md5sums.txt files due to a mistake in which `BS_E0AAK0V0` was in the patient-id column.

**Comment by jaclyn-taroni** [comment]
The new `pbta-histologies.tsv` file is in the testing bucket on S3 and I've updated `md5sum.txt` there as well.

**Comment by jaclyn-taroni** [comment]
I cancelled that build -- the one currently running is here: https://circleci.com/gh/AlexsLemonade/OpenPBTA-analysis/1939

**Pull request merged**